webapp/packages/core-blocks/src/ObjectPropertyInfoForm/ObjectPropertyInfoForm.tsx

```diff
@@ -57,6 +59,14 @@ const RenderField: React.FC<RenderFieldProps> = observer(function RenderField({
     description = 'Password saved';
   }
 
+  if (!editable && property.value) {
+    return (
+      <FormFieldDescription label={property.displayName} raw>
+        {property.value}
+      </FormFieldDescription>
+    );
+  }
+
```

Comment:
why double-check? so if we set property `editable=false` but get property without value, we show input? what?